artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/perf/MicrosClock.java

```diff
@@ -43,23 +45,28 @@ public static boolean isAvailable() {
    }
 
    public static long now() {
-      long epochSecond = offset;
-      long nanoAdjustment = jdk.internal.misc.VM.getNanoTimeAdjustment(epochSecond);
+      try {
+         long epochSecond = offset;
+         Class vm = Class.forName("jdk.internal.misc.VM");
+         Method getNanoTimeAdjustment = vm.getMethod("getNanoTimeAdjustment", long.class);
+         long nanoAdjustment = (long) getNanoTimeAdjustment.invoke(getNanoTimeAdjustment, epochSecond);
```

Comment:
This might be better done once, e.g during the initial availability checks, and then the Method reused here rather than doing lookups every time, given the aimed usage ?

Reply:
Done.